Classes/Utility/RequestUtility.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/yellowpages2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Yellowpages2\Utility;
+
+/*
+ * Request Utility class
+ */
+
+use TYPO3\CMS\Core\Utility\ArrayUtility;
+
+class RequestUtility
```

Comment:
I don't like Utility classes, as they are damn hard to test, if they have relations. But as this class has no further relations, this class is OK ;-)